extensions/arc/src/common/utils.ts

```diff
@@ -107,20 +107,37 @@ export function getDatabaseStateDisplayText(state: string): string {
  * @param name The name of the resource to delete
  * @returns Promise resolving to true if the user confirmed the name, false if the input box was closed for any other reason
  */
-export async function promptForResourceDeletion(namespace: string, name: string): Promise<boolean> {
+
+/**
+ * Opens an input box prompting and validating the user's input.
+ * Used because vscode.window.showInputBox doens't let you set the title.
```

Comment:
@Charles-Gagnon At least that's why I didn't use showInputBox.  showInputBox lets you set a prompt beneath the input.  But not the title above the input.  The validation message overrides the area beneath the input, so the title is useful context to the operation you're performing.  Any other reason you didn't use showInputBox?

Reply:
Well, and showInputBox doesn't allow for the other stuff I wanted to do either like clearing the warning message when the user started typing again (it'd show the error message until the correct value was entered)